packages/admin/cms-admin/src/index.ts

```diff
@@ -37,6 +37,7 @@ export { useSaveState } from "./common/useSaveState";
 export { ContentScopeIndicator } from "./contentScope/ContentScopeIndicator";
 export type { ContentScopeControlsConfig } from "./contentScope/Controls";
 export { ContentScopeControls } from "./contentScope/Controls";
+export { LegacyContentScopeIndicator } from "./contentScope/LegacyContentScopeIndicator";
```

Comment:
Do we need this? Isn't the Theme Refactor a major breaking change anyway?

Reply:
I'm not sure if it's necessary to force all existing projects to migrate to the new toolbar. We also don't force them to migrate from Table to DataGrid (although that's obviously a bigger rewrite).

Maybe we can even keep the Toolbar compatible. If it's possible that would be nice. But I have yet to try this.